diff --git a/http/Provision/Service/http/public.php b/http/Provision/Service/http/public.php
index d35a361..486979d 100644
--- a/http/Provision/Service/http/public.php
+++ b/http/Provision/Service/http/public.php
@@ -114,7 +114,7 @@ class Provision_Service_http_public extends Provision_Service_http {
   function grant_server_list() {
     return array(
       $this->server,
-      $this->context->platform->server,
+      isset($this->context->wpplatform) ? $this->context->wpplatform->server : $this->context->platform->server,
     );
   }
 }
